fixups/FileRedirectionFixup/PathRedirection.cpp

```diff
@@ -274,16 +276,49 @@ void LogString(const char* name, const wchar_t* value)
 }
 void LogString(const wchar_t* name, const char* value)
 {
-    Log(L"%ls=%s\n", name, value);
+    if ((value != NULL && value[1] != 0x0))
+        Log(L"%ls=%s\n", name, value);
+    else
+        Log(L"%ls=%ls", name, (wchar_t*)value);
 }
 void LogString(const wchar_t* name, const wchar_t* value)
 {
-    Log(L"%ls=%ls\n", name, value);
+    if ((value != NULL && ((char*)value)[1] == 0x0))
+        Log(L"%ls=%ls\n", name, value);
+    else
+        Log(L"%ls=%s", name, (char*)value);
+}

```

Comment:
Please add one blank line after each method.

Reply:
Addressed in Commit 1bdc01a3